src/eei.h

```diff
@@ -50,8 +51,22 @@ class InternalErrorException : std::exception {
   if (!(condition)) throw InternalErrorException(msg); \
 }
 
+struct ExecutionResult {
+  uint64_t gasLeft;
+  std::vector<char> returnValue;
+};
+
 struct EthereumInterface : ShellExternalInterface {
-  EthereumInterface(Hera const& _hera, HeraCall & _call) : ShellExternalInterface(), hera(_hera), call(_call) { (void)hera; }
+  EthereumInterface(
+    struct evm_context const& _context,
+    struct evm_message const& _msg,
+    ExecutionResult & _result
+  ):
+    ShellExternalInterface(),
+    context(_context),
+    msg(_msg),
+    result(_result)
+  { (void)context; }
```

Comment:
The `(void)` is not needed.

Reply:
Needed to silence unused variable warning (soon it will be used).